refractr/refractr.yml

```diff
@@ -111,75 +110,86 @@ refracts:
 #				  specified. However, the 'tests' must be specified for rewrites
 #				  such as this.
 
+refracts:
+
 - srcs:
   - firefox.com
   - www.firefox.com
   dst:
     # bug 1485429, 1432228
-  - /switch: https://www.mozilla.org/en-US/firefox/
-  - /mobile: https://www.mozilla.org/mobile/
+  - /switch: www.mozilla.org/firefox/switch/?redirect_source=firefox-com
 
-  - /beta: https://www.mozilla.org/firefox/channel/
```

Comment:
Don't know but from somewheres.  I didn't invent it.  If it truly doesn't exist we can remove it.  Keep in mind, many months have passed since you and I moved this stuff over.  Could have been decom'd from the apache in the interim.